pkg/aws/disk.py

```diff
@@ -12,22 +13,30 @@ async def start_cloning(project, hostname):
     con = create_db_con()
     try:
         bucket = Bucket.objects(project=project).allow_filtering()[0]['bucket']
-        accesskey = Bucket.objects(project=project).allow_filtering()[0]['access_key']
+        access_key = Bucket.objects(project=project).allow_filtering()[0]['access_key']
         secret_key = Bucket.objects(project=project).allow_filtering()[0]['secret_key']
         public_ip = Discover.objects(project=project,host=hostname).allow_filtering()[0]['public_ip']
         user = Project.objects(name=project).allow_filtering()[0]['username']
     except Exception as e:
         print("Error occurred: "+str(e))
     load_dotenv()
+    username = Project.objects(name=project)[0]['username']
     mongodb = os.getenv('BASE_URL')
-    current_dir = os.getcwd()
-    print("/usr/local/bin/ansible-playbook -i "+current_dir+"/ansible/"+project+"/hosts "+current_dir+"/ansible/aws/start_migration.yaml -e \"bucket="+bucket+" access_key="+accesskey+" secret_key="+secret_key+" mongodb="+mongodb+ " project="+project+"\"")
+    current_dir = os.getcwd() 
+    playbook = "start_migration.yaml"
+    stage = "start clone"
```

Comment:
Use start_clone. But cloning might be better as the log file would appear as cloning _log.txt then